### ISSUE TITLE
Tab buttons at top aren’t clickable with iOS Voice Control on an iPhone

### ISSUE THREAD

**Issue by iliana** [nolanlawson/pinafore#1735]
Filing this issue on behalf of a friend, who can’t access the tabs at the top since they don’t register as a tappable/clickable area.

This might be happening because the tabs don’t have text displayed due to screen width. The tappable areas are detected on an iPad.

Post with video and context is at https://mst3k.interlinked.me/@artemis/103933244302642043

> When I say "show numbers" all tappable things on screen should get a number and I can say a number to tap it. but the bug is the nav bar items don't get numbers. Same for "show names" but names come from aria labels or tries to guess if they're missing
>
> The url is dev but it happens on dev and not dev

**Comment by nolanlawson** [comment]
Thanks for filing the bug! Those buttons definitely have `aria-label`s, so I wonder what the issue is. It looks like they're using built-in voice navigation on an iPhone, so I ought to be able to repro that way.
